Simplenote/src/main/java/com/automattic/simplenote/models/Note.java

```diff
@@ -144,6 +148,15 @@ public static Calendar numberToDate(Number time) {
         return date;
     }
 
+    public static String numberToDateString(@NonNull Number number) {
+        long milliseconds = new BigDecimal(number.toString()).multiply(new BigDecimal(1000)).longValue();
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTimeInMillis(milliseconds);
+        SimpleDateFormat date = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US);
+        date.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return date.format(calendar.getTime());
```

Comment:
looks like we may be able to use `Instant` if it's available

```java
Instant.ofEpochSecond(number).toString() // formatted in ISO8601 format
```

Reply:
Nice find!  [`Instant`](https://developer.android.com/reference/java/time/Instant) requires API 26 and our minimum SDK version is 23.  Once we updated `minSdkVersion` to 26, we should definitely use `Instant.ofEpochSecond(number).toString()`.